src/Compilers/Test/Core/ExceptionHelper.cs

```diff
@@ -25,11 +25,12 @@ internal static string GetMessageFromResult(IEnumerable<Diagnostic> diagnostics,
             return sb.ToString();
         }
 
-        internal static string GetMessageFromResult(string output, string exePath)
+        internal static string GetMessageFromResult(string output, string exePath, bool isIlVerify = false)
         {
             StringBuilder sb = new StringBuilder();
             sb.AppendLine();
-            sb.Append("PeVerify failed for assembly '");
+            string tool = isIlVerify ? "IlVerify" : "PeVerify";
```

Comment:
Nit: generally they use the first two letters are upper case

```suggestion
            string tool = isIlVerify ? "ILVerify" : "PEVerify";
```